e2ev2/tests/workspace-create.test.js

```diff
@@ -51,14 +51,15 @@ browserTest('create a workspace', async browser => {
   const workspacePageTitleLink =
     await page.waitForSelector('a[href="/workspaces"] + span + div > a')
   await expect(workspacePageTitleLink.evaluate(e => e.innerText)).resolves.toBe(wsName)
-  await page.click('[aria-label="Open Actions Menu"]')
+  await page.waitForSelector('[aria-label="Open Actions Menu"]').then(eh => eh.click())
```

Comment:
nice.  I would bet that there are many places we could apply this to increase reliability.  (whether we should continue to invest in these tests is a separate question)